build-tools/automation/azure-pipelines.yaml

```diff
@@ -0,0 +1,213 @@
+        cp bin/Build$(XA.Build.Configuration)/*.pkg $(System.DefaultWorkingDirectory)/unsigned-installers
+      displayName: copy build artifacts
+
+    - task: PublishPipelineArtifact@0
+      displayName: upload unsigned installers
+      inputs:
+        artifactName: unsigned
+        targetPath: $(System.DefaultWorkingDirectory)/unsigned-installers
+
+    - task: MSBuild@1
+      displayName: package build results
+      inputs:
+        solution: build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks.csproj
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: /t:ZipBuildStatus /p:ResultPackageOutputPath=$(Build.ArtifactStagingDirectory)
+      condition: always()
+
+    - task: PublishPipelineArtifact@0
+      displayName: upload artifacts
+      inputs:
+        artifactName: mac-build-results
+        targetPath: $(Build.ArtifactStagingDirectory)
+      condition: always()
+
+- stage: win_build_test
+  displayName: Win
+  dependsOn: prepare
+  jobs:
+  - job: build_test
+    displayName: Build and Test
+    pool: VSEng-MicroBuildVS2017
+    timeoutInMinutes: 360
+    cancelTimeoutInMinutes: 5
+    steps:
+    - task: DownloadPipelineArtifact@1
+      inputs:
+        artifactName: $(BundleArtifactName)
+        downloadPath: $(System.DefaultWorkingDirectory)
+
+    - task: MSBuild@1
+      displayName: msbuild Xamarin.Android /t:Prepare
+      inputs:
+        solution: Xamarin.Android.sln
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: /t:Prepare /bl:bin\Build$(Configuration)\msbuild-prepare.binlog /p:BundleRootPath=$(System.DefaultWorkingDirectory)
+
+    - task: MSBuild@1
+      displayName: msbuild Xamarin.Android /t:Build
+      inputs:
+        solution: Xamarin.Android.sln
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: /t:Build /bl:bin\Build$(Configuration)\msbuild-build.binlog /p:BundleRootPath=$(System.DefaultWorkingDirectory)
+
+    - task: MSBuild@1
+      displayName: msbuild create-vsix
+      inputs:
+        solution: build-tools\create-vsix\create-vsix.csproj
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: /p:CreateVsixContainer=True /p:ZipPackageCompressionLevel=Normal /bl:bin\Build$(Configuration)\msbuild-create-vsix.binlog
+
+    - task: CmdLine@1
+      displayName: xabuild Xamarin.Android-Tests
+      inputs:
+        filename: bin\$(XA.Build.Configuration)\bin\xabuild.exe
+        arguments: Xamarin.Android-Tests.sln /p:Configuration=$(XA.Build.Configuration) /p:XAIntegratedTests=False /bl:bin\Build$(Configuration)\msbuild-build-tests.binlog
+
+    - task: MSBuild@1
+      displayName: nunit Xamarin.Android.Build.Tests
+      inputs:
+        solution: Xamarin.Android.sln
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: /t:RunNUnitTests /bl:bin\Build$(Configuration)\msbuild-run-nunit-tests.binlog
```

Comment:
Test logs should go into `bin/Test$(Configuration)`.

Reply:
This should be resolved in the latest commit.